### PR TITLE
Fix tests errors for rhel-devel

### PR DESCRIPTION
**Pull request by jkonecny12**
These are issues which needs to be resolved on rhel-devel to make tests working again.

### CODE REVIEW

**Comment by jkonecny12** [comment]
**UPDATED**
* fix rpm test issue

File: tests/nosetests/pyanaconda_tests/ks_version_test.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ class CommandVersionTestCase(unittest.TestCase):
 
     # Names of the kickstart commands and data that should be temporarily ignored.
     IGNORED_NAMES = {
+        "btrfs"
```
Comment (poncovka) [inline tests/nosetests/pyanaconda_tests/ks_version_test.py]: You should just update the version of the command btrfs to `RHEL8_BTRFS` in `pyanaconda/core/kickstart/commands.py`.

**Comment by jkonecny12** [comment]
**UPDATED**:
* Implemented correct fix for BTRFS error

Thanks for pointing that out @poncovka . I thought that this is a problem thanks to the removal.

**Comment by jkonecny12** [comment]
**UPDATED**:
* Fix new errors in `ihelp.py`
* Add missing pylint false positive
* one line code polishing

**Pull request merged**